PowerShell/JumpCloud Module/Public/RadiusServer/Set-JCRadiusServer.ps1

```diff
@@ -30,9 +53,21 @@ Function Set-JCRadiusServer {
         ($FunctionParameters).Add('Action', $Action) | Out-Null
         ($FunctionParameters).Add('Type', $Type) | Out-Null
         # Run the command
+        Write-Host $FunctionParameters.keys
```

Comment:
Write-Host? do we really want to be writing this out?

Reply:
Was this used for debugging purposes? If so then we can just remove it